modules/install/pages/install.adoc

```diff
@@ -1,71 +1,538 @@
+* This method is preferred when:
+** You don't have GUI support for the EAP 8 installation.
+** You don't want to have any custom configuration.
+** You need text based console for the EAP 8 installation.
+* Due to its portable nature, you can use it on multiple platforms(like Windows, Linux, etc.) and it requires OpenJDK 17 installed on the host OS.
+
+== Installing JBoss EAP using archive installation method
+
+* In this method, the JBoss EAP 8 is distributed as an archive file that you can extract and execute on an operating system (OS) with  OpenJDK 17 installed.
+* The archive file contains all the libraries, configuration files, and scripts needed to start JBoss EAP, *without* further customization.
+* Just like zip file method, this method is preferred when:
+** You don't have GUI support for the EAP 8 installation.
+** You don't want to have any custom configuration.
+** You need text based console for the EAP 8 installation.
+* Due to its portable nature, you can use it on multiple platforms(like Windows, Linux, etc.) and it requires OpenJDK 17 installed on the host OS.
+
+== Installing JBoss EAP by using the RPM installation method
+
+* JBoss EAP 8 is available as an RPM package for users with a valid subscription to the JBoss Enterprise Application Platform sub-channel, in the JBoss Enterprise Application Platform group.
+* The yum command downloads all the RPMs to the host OS, and automatically installs all the files needed to start and manage JBoss EAP.
+* The RPM installation method is only valid for Red Hat Enterprise Linux (RHEL) operating system and not for Windows server.
+* This method is preferred when you want JBoss EAP service files to be installed for the RHEL operating system.
 
 == Installing JBoss EAP using the JAR installer
 
-Brief overview only. Details in lab section below....
+* This method can also be called the GUI installer method.
+* In this method, the jar file is a Java based application that is responsible for providing a step-by-step way to customize the environment setup.
+* This method is preferred when:
+** You want to install on *multiple platforms* like Windows server, Linux etc.
+** You need either graphical interface or a text based console.
+** You want to customise JBoss EAP installation.
+* Due to its flexible nature, the installer can install JBoss EAP by using a response file, with all the options passed as an XML file.
 
-== Installing JBoss EAP using Installation Manager
+== Pre-requisites
 
-Brief overview only. Details in lab section below....
+* You have access to the internet.
+* You have reviewed https://access.redhat.com/articles/6961381[JBoss EAP 8.0 supported configurations].
+* You have created an account on the https://access.redhat.com/[Red Hat Customer Portal] to download JBoss EAP files.
+* You are operating a RHEL 9 server equipped with a graphical user interface (GUI) package.
```

Comment:
This should not be mandatory. This is mandatory only if you need to run the graphical JAR installer in GUI mode.

Reply:
Removed these pre-requisites